game-core/src/main/java/games/strategy/engine/framework/lookandfeel/LookAndFeel.java

```diff
@@ -41,11 +42,24 @@
 import games.strategy.debug.ClientLogger;
 import games.strategy.engine.framework.system.SystemProperties;
 import games.strategy.triplea.settings.ClientSetting;
+import games.strategy.triplea.settings.SettingsWindow;
 
 /**
  * Provides methods for working with the Swing Look-And-Feel.
  */
 public final class LookAndFeel {
+  static {
+    ClientSetting.LOOK_AND_FEEL_PREF.addSaveListener(newValue -> {
+      setupLookAndFeel(newValue);
+      SettingsWindow.updateLookAndFeel();
+      JOptionPane.showMessageDialog(
+          null,
+          "WARNING: look and feel changes can cause instability. Please restart all running TripleA instances",
+          "Close TripleA and Restart",
+          JOptionPane.WARNING_MESSAGE);
+    });
+  }
+
```

Comment:
note, this is an extra warning message asking user to restart:
![warning_message](https://user-images.githubusercontent.com/12397753/36942077-6fa1d820-1f1f-11e8-9e8a-0af40f9e3fe9.png)

After closing you get the standard property changed pop-up.